app/api/search/route.tsx

```diff
@@ -0,0 +1,8 @@
+import { NextResponse } from "next/server";
+import searchSchema from "../../src/data/schemas/searchSchema";
+
+export const GET = async () => {
+  return NextResponse.json(searchSchema, { status: 200 });
```

Comment:
I'm not sure I understand how `status` is working in this setup. Does this mean it would live on the response (like we've been talking about) instead of within the `headers` field